### PR TITLE
[#663] ChartView: add total contribution of group

### LINKED ISSUE #663: Show total contribution of a group
It may be useful to show the total contribution of a group.

image

### PR DESCRIPTION
**Pull request by jamessspanggg**
Fixes #663 

```
A group only shows the total contribution of each author or repo/branch
within the group but not the total contribution of all combined.

It is rather difficult for the user to compare the total contribution
between groups as the user would need to sum up the contribution of each
row in order to get the total contribution of a group.

Let's display the total contribution of groups beside the group title so
that it is easier for the user to do comparisons between groups.
```

### CODE REVIEW

**Comment by jamessspanggg** [comment]
Sorry for recreating the PR. I was trying to get rid of the unnecessary commits.

**Comment by fzdy1914** [comment]
Oh, just `rebase -i` and force push will do.

**Comment by fzdy1914** [comment]
For further review, do notice the hovering text.

File: frontend/src/static/js/v_summary.js [hunk 1]
```diff
@@ -543,6 +543,14 @@ window.vSummary = {
     getGroupCommitsVariance(total, group) {
       return total + group[this.sortingOption];
     },
+
+    getGroupTotalContribution(group) {
+      let totalContribution = 0;
+      group.forEach((ele) => {
+        totalContribution += ele.totalCommits;
```
Comment (eugenepeh) [inline frontend/src/static/js/v_summary.js]: can use `.reduce` to make it a 1 line solution.

Comment (jamessspanggg) [reply frontend/src/static/js/v_summary.js]: done

**Comment by eugenepeh** [comment]
>Oh, just rebase -i and force push will do.

Please don't create new PR / force push as we want to keep track of your changes, as well as comments, between reviews.

**Pull request merged**